CHANGELOG.md

```diff
@@ -1,106 +1,138 @@
 # Change Log
 
+## v1.0.37 (2022-09-21)
+
+- Add `permalink` to `AlertGroup` public API response schema
+- Update `DEVELOPER.md` to reflect successfully running project locally for the first time
```

Comment:
Not sure if we want to include this to changelog since we only include user-facing changes there

Reply:
@vadimkerr it's public API change, so sounds pretty user-facing for me :)

Reply:
just to clarify, you mean just the line regarding the `DEVELOPER.md` change, correct?

Reply:
yup, only the second point about DEVELOPER.md